use-cases.md

```diff
@@ -29,13 +30,19 @@ Performed periodically after the connection is set up so that RMS has future res
 
 To know the data about the rates of the enterprise, there are two relevant operations. [Get all rates](operations/services.md#get-all-rates) can give you information about the names \(and ids\) of the rates in the property, their status, rate groups and restrictions. [Get rate pricing](operations/services.md#get-rate-pricing) gives you the pricing of specific rate for a specific time period. In order to update rate prices, [Update rate price](operations/services.md#update-rate-price) operation be used. Individual rate, room category and time span can be chosen.
 
+### Restrictions
+
+To retrieve all restrictions active in the enterprise, the RMS should periodically use the [Get all restrictions](operations/services.md#get-all-restrictions) endpoint which will return information about the applicability of the restriction.
```

Comment:
Add some definition of restriction? We have a definition of the revenue management system above.